awscli/autoprompt/autoprompt_driver.py

```diff
@@ -10,32 +10,9 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
```

Comment:
Let's just rename this file to `core` it make it shorter, a little less redundant, and encapsulate that the file holds the main utilities for auto-prompt and not necessarily just the driver.